js/logging.js

```diff
@@ -139,6 +139,10 @@ loggingModule.factory(
                 }
             }
 
+            if (LOGGING_CONFIG.FORWARD_TO_NEWRELIC && $window.NREUM && $window.NREUM.noticeError) {
+                $window.NREUM.noticeError(message, {desc: desc});
```

Comment:
If we pass in a `desc` this currently goes to logstash, I'm now passing that into `noticeError` function, which supports custom attributes, so we also get it in NewRelic. But for reasons known only to NewRelic that information only appears in the Insights explorer and not in the error reports we get in the Browser module. 

_shakes fist at NewRelic_

Reply:
Does this mean we aren't going to see all our errors in the same place?

Reply:
All the errors will still be in the errors page in NewRelic, but any additional context to the error we pass into through `desc` will only be available through their Insights module.

Reply:
That's ok, we can create a custom dashboard with insights, so we could create one for error reports to get more detail.